tests/test-cases/Group3-Docker-Compose/3-03-Docker-Compose-Basic.robot

```diff
@@ -73,6 +79,9 @@ Compose Up with link
     Should Be Equal As Integers  ${rc}  0
     Should Contain  ${output}  PING aaa
```

Comment:
is it viable to have "ping -c 3 aaa" or similar as the command so that the ping container exits, then wait on that container? @kreamyx has pointed out that this is potentially racy - while it should be rare I think he is correct.